src/commands/anime/studioghibli/character.js

```diff
@@ -0,0 +1,45 @@
+const { Command, SwitchbladeEmbed } = require('../../')
+
+const fetch = require('node-fetch')
+
+module.exports = class Character extends Command {
+  constructor (client) {
+    super(client, {
+      name: 'character',
+      parentCommand: 'studioghibli'
+    })
+  }
+
+  async run ({ t, author, channel }, searchTerm) {
+    const embed = new SwitchbladeEmbed(author)
+    channel.startTyping()
+    const body = await fetch('https://ghibliapi.herokuapp.com/people').then(
+      res => res.json()
+    )
+    const characterData = body.filter(
+      element => element.name === searchTerm
+    )[0]
+
+    if (characterData === 'undefined') {
+      embed.setImage(t('commands:studioghibli.character.notFound'))
```

Comment:
Why are you setting the image to text? That error text should be either the embed title or description, like so:
```js
embed.setDescription(t('commands:studioghibli.character.notFound'))
```

Actually, we have a thing called `CommandErrors`. They're throwable and require less code on the command. After including the class on the first line, you'd be able to use
```suggestion
      throw new CommandError(t('commands:studioghibli.subcommands.character.notFound'))
```